qml-ui/pages/SessionDashboard/Main.qml

```diff
@@ -217,6 +218,18 @@ Zynthian.ScreenPage {
         }
     }
 
+    Connections {
+        target: zynthian
+        onCurrent_screen_idChanged: {
```

Comment:
I'm not sure this should be done at screen id change:
the proper layer should always be selected and not have to change page in order to fix the selection?

Reply:
This is to ensure when a user first invokes the layerPopup from bottom drawer, clicks on new synth and then clicks back to get back to home page and doesnt create a synth.

In that case the layer is already switched to an empty layer where the new synth would have been created but since the user pressed back, it needs to switch to the selected track's connected sound if any.

Although any better alternative than listening to page change would be great.